ncm-spma/src/main/bin/spma

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+touch /.spma-run
```

Comment:
This is a simple way of implementing the "run" functionality without needing a separate code base. The component checks if that file exists and only executes the package changes if it does. (The code currently does not honour the "run" attribute - that will be fixed shortly.